slack/channels.md

```diff
@@ -90,6 +90,13 @@ Please note that like any open source project, the main language on this Slack i
 - Status: Public
 - Language: EN
 
+**#qa-core**
+- Description: For core maintainers and QA team, to discuss QA topics
```

Comment:
I really still don't understand why we reference private channels here :thinking:

Reply:
I dont know 🤔 I'm just following the rules

Reply:
ping @eternoendless @ttoine Is there a reason?